### PR TITLE
[FLINK-20458][docs] translate gettingStarted.zh.md and correct spelling errors in gettingStarted.md

### PR DESCRIPTION
**Pull request by jjiey**
## What is the purpose of the change

[FLINK-20458](https://issues.apache.org/jira/browse/FLINK-20458?src=confmacro)
Translate the doc "docs/dev/table/sql/gettingStarted.zh.md"

## Brief change log

  - Translate the doc "docs/dev/table/sql/gettingStarted.zh.md"
  - Correct spelling errors in "docs/dev/table/sql/gettingStarted.md"

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not documented

### CODE REVIEW

**Comment by jjiey** [comment]
Hi, @wuchong , Could you help to review this PR? thanks very much.
by the way, I am a newcomer, must this issue be assigned to me before I translate?

**Comment by wuchong** [comment]
@fsk119 Could you help to review this?

**Comment by wuchong** [comment]
@jjiey, please comment in the JIRA issue, so that I can assign the issue to you. Please get assigned before opening a pull request next time.

**Comment by jjiey** [comment]
> @jjiey, please comment in the JIRA issue, so that I can assign the issue to you. Please get assigned before opening a pull request next time.

got it. thanks very much! @wuchong

File: docs/dev/table/sql/gettingStarted.zh.md [hunk 1]
```diff
@@ -79,16 +79,15 @@ SELECT CURRENT_TIMESTAMP;
...
+可以通过 SQL 客户端或使用环境配置文件来定义表。SQL 客户端支持类似于传统 SQL 的 [SQL DDL 命令]({% link dev/table/sql/index.zh.md %})。标准 SQL DDL 用于[创建]({% link dev/table/sql/create.zh.md %})，[修改]({% link dev/table/sql/alter.zh.md %})，[删除]({% link dev/table/sql/drop.zh.md %})表。
 
-Flink has a support for different [connectors]({% link dev/table/connect.zh.md %}) and [formats]({%link dev/table/connectors/formats/index.zh.md %}) that can be used with tables. Following is an example to define a source table backed by a [CSV file]({%link dev/table/connectors/formats/csv.zh.md %}) with `emp_id`, `name`, `dept_id` as columns in a `CREATE` table statement.
+Flink 支持可以与表一起使用的不同[连接器]({% link dev/table/connect.zh.md %})和[格式]({% link dev/table/connectors/formats/index.zh.md %})。下面是一个示例，定义一个[CSV 文件]({% link dev/table/connectors/formats/csv.zh.md %})作为 source 表，其中 `emp_id`，`name`，`dept_id` 作为 `CREATE` 表语句中的列。
```
Comment (fsk119) [inline docs/dev/table/sql/gettingStarted.zh.md:90]: “Flink has a support for different [connectors] and [formats] that can be used with tables.” -> "Flink 支持 不同的 连接器和格式相结合以定义表。" 这样子翻译不知道准不准确？

"a source table backed by a CSV file" -> "以 CSV 文件作为存储格式的 source 表"

File: docs/dev/table/sql/gettingStarted.zh.md [hunk 2]
```diff
@@ -129,11 +128,11 @@ FROM employee_information
...
  {% endhighlight %} 
 
-Such queries are considered _stateful_. Flink's advanced fault-tolerance mechanism will maintain internal state and consistency, so queries always return the correct result, even in the face of hardware failure. 
+这样的查询被认为是 _有状态的_。Flink 的高级容错机制将维持内部状态和一致性，因此即使遇到硬件故障，查询也始终返回正确结果。
 
-## Sink Tables
+## Sink 表
 
-When running this query, the SQL client provides output in real-time but in a read-only fashion. Storing results - to power a report or dashboard - requires writing out to another table. This can be achieved using an `INSERT INTO` statement. The table referenced in this clause is known as a sink table. An `INSERT INTO` statement will be submitted as a detached query to the Flink cluster. 
+当运行此查询时，SQL 客户端实时但是以只读方式提供输出。存储结果（为报表或仪表板提供数据来源）需要写到另一个表。这可以使用 `INSERT INTO` 语句来实现。本节中引用的表称为 sink 表。`INSERT INTO` 语句将作为一个独立查询被提交到 Flink 集群中。
```
Comment (fsk119) [inline docs/dev/table/sql/gettingStarted.zh.md:135]: "存储结果（为报表或仪表板提供数据来源）需要写到另一个表" -> "存储结果，作为报表或仪表板提供数据来源，需要写到另一个表"

**Review by fsk119 — CHANGES_REQUESTED** [review]
Thanks for your contribution. Sorry for the late response and I left some comments.

File: docs/dev/table/sql/gettingStarted.zh.md (hunk 2 above)
Comment (jjiey) [reply docs/dev/table/sql/gettingStarted.zh.md:135]: “作为报表或仪表板提供数据来源” -> "作为报表或仪表板的数据来源"   理解您的意思了，这里“提供”改为“的”应该更通顺一些。

**Comment by jjiey** [comment]
> Thanks for your contribution. Sorry for the late response and I left some comments.

Thanks @fsk119 for your review comments. I've updated and commited the modifications.

File: docs/dev/table/sql/gettingStarted.zh.md [hunk 3]
```diff
@@ -129,11 +128,11 @@ FROM employee_information
...
+这样的查询被认为是 _有状态的_。Flink 的高级容错机制将维持内部状态和一致性，因此即使遇到硬件故障，查询也始终返回正确结果。
 
-## Sink Tables
+## Sink 表
 
-When running this query, the SQL client provides output in real-time but in a read-only fashion. Storing results - to power a report or dashboard - requires writing out to another table. This can be achieved using an `INSERT INTO` statement. The table referenced in this clause is known as a sink table. An `INSERT INTO` statement will be submitted as a detached query to the Flink cluster. 
+当运行此查询时，SQL 客户端实时但是以只读方式提供输出。存储结果，作为报表或仪表板的数据来源，需要写到另一个表。这可以使用 `INSERT INTO` 语句来实现。本节中引用的表称为 sink 表。`INSERT INTO` 语句将作为一个独立查询被提交到 Flink 集群中。
 
  {% highlight sql %}
  INSERT INTO department_counts
```
Comment (fsk119) [inline docs/dev/table/sql/gettingStarted.zh.md:138]: Please remove this blank space.
image

File: docs/dev/table/sql/gettingStarted.zh.md [hunk 4]
```diff
@@ -113,13 +112,13 @@ SELECT * from employee_information WHERE DeptId = 1;
...
+虽然最初设计时没有考虑流语义，但 SQL 是用于构建连续数据流水线的强大工具。Flink SQL 与传统数据库查询的不同之处在于，Flink SQL 持续消费到达的行并对其结果进行更新。
 
-A [continuous query]({% link dev/table/streaming/dynamic_tables.zh.md %}#continuous-queries) never terminates and produces a dynamic table as a result. [Dynamic tables]({% link dev/table/streaming/dynamic_tables.zh.md %}#continuous-queries) are the core concept of Flink's Table API and SQL support for streaming data. 
+一个[连续查询]({% link dev/table/streaming/dynamic_tables.zh.md %}#continuous-queries)永远不会终止，并会产生一个动态表作为结果。[动态表]({% link dev/table/streaming/dynamic_tables.zh.md %}#continuous-queries)是 Flink 中 Table API 和 SQL 对流数据支持的核心概念。
 
-Aggregations on continuous streams need to store aggregated results continuously during the execution of the query. For example, suppose you need to count the number of employees for each department from an incoming data stream. The query needs to maintain the most up to date count for each department to output timely results as new rows are processed.
+连续流上的聚合需要在查询执行期间不断地存储聚合的结果。例如，假设你需要从传入的数据流中计算每个部门的员工人数。查询需要维护每个部门最新的计算总数，以便在处理新行时及时输出结果。
 
  {% highlight sql %}
  SELECT
```
Comment (fsk119) [inline docs/dev/table/sql/gettingStarted.zh.md:124]: Please delete empty space before `SELECT`
image

**Review by fsk119 — CHANGES_REQUESTED** [review]
LGTM except the minor problems.

Please rebase the branch to the master and force push to the repo again. (Merge command is not recommended in the community)

**Comment by jjiey** [comment]
> LGTM except the minor problems.
> 
> Please rebase the branch to the master and force push to the repo again. (Merge command is not recommended in the community)

got it. Thanks @fsk119 for your guidance!

**Review by fsk119 — APPROVED** [review]
LGTM

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**